oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -589,6 +608,10 @@ public ServiceAccountImpersonationOptions getServiceAccountImpersonationOptions(
     return serviceAccountImpersonationOptions;
   }
 
+  protected String getMetricsSource() {
```

Comment:
getMetricsSource() is kind of confusing.

getCredentialSourcingMethodForMetrics?

getCredentialSource()?

Reply:
change to getCredentialSourceType()